config/manifests/bases/clusterserviceversion.yaml.in

```diff
@@ -5,6 +5,7 @@ metadata:
     alm-examples: '[]'
     capabilities: Basic Install
     operators.operatorframework.io/operator-type: non-standalone
+    olm.skipRange: @SKIP_RANGE@
```

Comment:
got it. `operators.operatorframework.io/operator-type: non-standalone` only downstream change

Reply:
Yes :+1: